product_multi_image/hooks.py

```diff
@@ -0,0 +1,21 @@
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+try:
+    from openerp.addons.base_multi_image.hooks import (
```

Comment:
Openerp - > odoo

Reply:
Thanks for your reviewing. It has been updated.